spec/specs_spec.rb

```diff
@@ -293,6 +293,11 @@ def get_true_hash_size(hash, obj_class)
     $dut.has_spec?(:tnikhov2).should == true
     $dut.has_spec?(/tnikhov/).should == true
   end
+
+  it "finding specs via :symbol option works" do
+    $dut.has_spec?(nil, symbol: :gvdd).should == true  
```

Comment:
@ginty
Yep, I like it.  I’ll make the change.


From: Stephen McGinty [mailto:notifications@github.com]
Sent: Wednesday, April 06, 2016 2:18 PM
To: Origen-SDK/origen <origen@noreply.github.com>
Cc: Aaron Burgmeier <aaron.burgmeier@nxp.com>
Subject: Re: [Origen-SDK/origen] Added capability to find specs using :symbol attribute (#45)


In spec/specs_spec.rb<https://github.com/Origen-SDK/origen/pull/45#discussion_r58765285>:

> @@ -293,6 +293,11 @@ def get_true_hash_size(hash, obj_class)

>      $dut.has_spec?(:tnikhov2).should == true

>      $dut.has_spec?(/tnikhov/).should == true

>    end

> +

> +  it "finding specs via :symbol option works" do

> +    $dut.has_spec?(nil, symbol: :gvdd).should == true

I think this would be more idiomatic Origen:

$dut.has_spec?(:gvdd, symbol: true)

What do you think?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Origen-SDK/origen/pull/45/files/d76e386893703f359e8e762090787309f915a26b#r58765285>